contrib/babelfishpg_common/sql/upgrades/babelfish_common_helper--4.3.0--4.4.0.sql

```diff
@@ -63,6 +63,9 @@ WITH FUNCTION sys.varchar2time(sys.VARCHAR, INT4) AS IMPLICIT;
 
 CALL sys.babelfish_drop_deprecated_object('function', 'sys', 'varchar2time_deprecated_4_4_0');
 
+CREATE CAST (bool AS sys.BIT)
+WITHOUT FUNCTION AS IMPLICIT;
+
```

Comment:
As these performance improvements will be part of 3_X_DEV as well, upgrading from 3.8.0 to 4.4.0 will cause failures due to the existing cast. Can we add a check to create the cast only if it doesn't already exist?

Reply:
Added